docs/layers.md

```diff
@@ -210,6 +210,7 @@ Values for `kind_detail`  are sourced from OpenStreetMap's `building` tag for bu
 * `addr_housenumber`: value from OpenStreetMap's `addr:housenumber` tag
 * `addr_street`: value from OpenStreetMap's `addr:street` tag
 * `area`: in square meters (spherical Mercator, no real-world), `polygon` features only. _See planned bug fix in [#1095](https://github.com/tilezen/vector-datasource/issues/1095)._
+* `building_material`: A description of the material covering the outside of the building or building part, if the information is available. Common values are: `cement_block`, `brick`, `plaster`, `wood`, `concrete`, `metal`, `stone`, `traditional`, `glass`, `mud`, `masonry`, `steel`, `semi-permanent`, `tin`, `permanent`, `timber-framing`, `sandstone`, `clay` and `other`, and there are many other less common values.
```

Comment:
Generally we alphabetize these sorts of lists.

Reply:
This was in order of the number of uses, descending. I thought that would be more useful to anyone who only wants to support the top N values. Would it be better to call that out, or alphabetise the list and let people pick values they think are important?

Reply:
Yes, please sort a-z.